gremlin-python/src/main/python/gremlin_python/process/traversal.py

```diff
@@ -843,7 +843,7 @@ def _arg_as_string(self, arg):
                 return tmp
 
         if isinstance(arg, Vertex):
-            return f'new ReferenceVertex({self._arg_as_string(arg.id)},\'{arg.label}\')'
+            return f'{self._arg_as_string(arg.id)}'
```

Comment:
Python implements the Vertex to id sugar at the GremlinLang level instead of special cases for V(), from(), to(), and mergeE(). Would we want to copy this same approach for other GLVs as they adopt GremlinLang or is it best to focus the sugar to specific steps?

Reply:
Opted to implement on an individual step bases